README.md

```diff
@@ -1,2 +1 @@
-# projectreview
-Microverse Trials Project Review
+Tribute Page by Angel and Eduardo.
```

Comment:
README file should be a lot more descriptive. There's no real information about what the project is about, how to use it, or how it looks like.

Reply:
- [ ] I suggest you to write comments kindly requesting changes:
✔️ * This is a polite suggestion: *"I think it would be better to make README file more descriptive"*
🚫 * This reads as an order: *"README file should be a lot more descriptive"*

- [ ] It would be nice to add a short explanation, a "why", for each requested change, it will help the project owner to understand why he/she needs to make the changes 👍🏻 smile

Reply:
Thank you! Will try again :smile: